mumctl/src/main.rs

```diff
@@ -223,6 +223,10 @@ impl fmt::Display for CliError {
 }
 
 fn main() {
+    if std::env::args().any(|s| s.as_str() == "--version" || s.as_str() == "-V") {
+        println!("mumctl {}", env!("VERSION"));
+        return;
+    }
```

Comment:
`clap` handles this for us.

Reply:
Not with our env-variable. This is the current behaviour.

```
$ cargo run --bin mumd -- --version
    Finished dev [unoptimized + debuginfo] target(s) in 0.05s
     Running `target/debug/mumd --version`
mumd v0.3-291-gd7a1738
$ cargo run --bin mumctl -- --version
    Finished dev [unoptimized + debuginfo] target(s) in 0.06s
     Running `target/debug/mumctl --version`
mumctl 0.4.0
```

Reply:
We have build scripts in both mumd and mumctl that use `git-describe` so the version number contains the hash.

Reply:
Maybe make which version you use toggleable behind a config flag? I agree that it's great to have the commit hash in development builds, but I don't think that that is something that is necessary for a final user. Maybe include the hash in debug builds but don't in release? Also, I think it's weird that mumd reports its version as `0.3` while mumctl reports its version as `0.4`

Reply:
> Maybe make which version you use toggleable behind a config flag? I agree that it's great to have the commit hash in development builds, but I don't think that that is something that is necessary for a final user.

Yes, that works. I'll hook something up.

> Also, I think it's weird that mumd reports its version as 0.3 while mumctl reports its version as 0.4.

That's what this PR fixes. With these changes both mumd and mumctl report the same version with the hash.

Reply:
> That's what this PR fixes. With these changes both mumd and mumctl report the same version with the hash.

:+1: